src/ui/components/Accordion.module.css

```diff
@@ -0,0 +1,46 @@
+.accordion {
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.accordionItemContainer:first-child {
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+  border-top: none;
+}
+
+.accordionItemContainer.open {
+  flex: 0 1 auto;
+  min-height: 20%;
+  overflow-y: scroll;
+}
+
+.accordionItemContainer {
+  border-top-width: 1px;
+}
+
+.accordionItemContainer:not(.open) {
+  flex-shrink: 0;
+}
+
+.accordionItemContainer .header {
+  background: white;
+  cursor: pointer;
+  font-size: 15px;
+  line-height: calc(16 / 12);
```

Comment:
Small nit but we can probably get away with just `1.25`.

Reply:
Yeah, this was pre-existing, but I agree with you, I'll change it 👍